examples/transfer/plot_bci_example.py

```diff
@@ -0,0 +1,199 @@
+"""
+====================================================================
+Motor imagery classification by transfer learning
+====================================================================
+
+Classify motor imagery data with transfer learning applied to MDM [1]_.
+"""
+
+import numpy as np
+from tqdm import tqdm
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import StratifiedShuffleSplit
+import matplotlib.pyplot as plt
+
+from mne import Epochs, pick_types, events_from_annotations
+from mne.io import concatenate_raws
+from mne.io.edf import read_raw_edf
+from mne.datasets import eegbci
+from mne import set_log_level
+
+from pyriemann.classification import MDM
+from pyriemann.estimation import Covariances
+from pyriemann.transfer import (
+    encode_domains,
+    TLDummy,
+    TLCenter,
+    TLClassifier,
+    TLSplitter,
+)
+
+set_log_level(verbose=False)
+
+
+###############################################################################
+
+def get_subject_dataset(subject):
+
+    # Consider epochs that start 1s after cue onset.
+    tmin, tmax = 1., 2.
+    event_id = dict(hands=2, feet=3)
+    runs = [6, 10, 14]  # motor imagery: hands vs feet
+
+    # Download data with MNE
+    raw_files = [
+        read_raw_edf(f, preload=True) for f in eegbci.load_data(subject, runs)
+    ]
+    raw = concatenate_raws(raw_files)
+
+    # Select only EEG channels
+    picks = pick_types(
+        raw.info, meg=False, eeg=True, stim=False, eog=False, exclude='bads')
+    # select only nine electrodes: F3, Fz, F4, C3, Cz, C4, P3, Pz, P4
+    picks = picks[[31, 33, 35, 8, 10, 12, 48, 50, 52]]
+
+    # Apply band-pass filter
+    raw.filter(7., 35., method='iir', picks=picks)
+
+    # Check the events
+    events, _ = events_from_annotations(raw, event_id=dict(T1=2, T2=3))
+
+    # Define the epochs
+    epochs = Epochs(
+        raw,
+        events,
+        event_id,
+        tmin,
+        tmax,
+        proj=True,
+        picks=picks,
+        baseline=None,
+        preload=True,
+        verbose=False)
+
+    # Extract the labels for each event
+    labels = epochs.events[:, -1] - 2
+
+    # Compute covariance matrices on scaled data
+    covs = Covariances().fit_transform(1e6 * epochs.get_data())
+
+    return covs, labels
+
+
+###############################################################################
+
+# We will consider subjects from the Physionet EEG database for which the
+# intra-subject classification has been checked to be > 0.70
```

Comment:
```suggestion
###############################################################################
# We will consider subjects from the Physionet EEG database for which the
# intra-subject classification has been checked to be > 0.70
```

otherwise it does not render as a clean section in the doc